cmr/queries.py

```diff
@@ -365,6 +385,22 @@ def bearer_token(self, bearer_token: str) -> Self:
 
         return self
 
+    def client_id(self, id_: str) -> Self:
```

Comment:
@Sharvin-M, sorry about the typing error. In the `__init__` method, keep the call as `self.client_id()` and simply update the functions signature as follows:

```suggestion
    def client_id(self, id_: Optional[str] = None) -> Self:
```